python/hsfs/feature_store.py

```diff
@@ -903,6 +918,70 @@ def get_transformation_function(
     ):
         """Get  transformation function metadata object.
 
+        !!! example "Get transformation function by name. This will default to version 1"
+            ```python
+
+            plus_one_fn = fs.get_transformation_function(name="plus_one")
+
+            ```
+
+        !!! example "Get built-in transformation function min max scaler"
+            ```python
+
+            min_max_scaler_fn = fs.get_transformation_function(name="min_max_scaler")
+
+            ```
+
+        !!! example "Get transformation function by name and version"   
+            ```python
+            
+            min_max_scaler = fs.get_transformation_function(name="min_max_scaler", version=2)
+
+            ```
+
+        You can define in the feature view transformation functions as dict, where key is feature name and value is online transformation function name.
+        Then the transformation functions are applied when you read training data, read batch data, or get feature vectors.
```

Comment:
`value is online transformation function name` this is not true it must be `value is online transformation function instance`